PROTOCOL.md

```diff
@@ -1933,6 +1939,283 @@ delta-table-version: 123
 }
 ```
 
+### Read Change Data Feed from a Table
+This is the API for clients to read change data feed from a table.
+
+You can provide either version or timestamp for the start and end. The start and end versions and timestamps are inclusive in the queries. To read the changes from a particular start version to the latest version of the table, specify only the starting version or timestamp.
```

Comment:
Mention that one of either starting version or starting timestamp must be provided? All those query parameters are marked as optional below.

Reply:
Added.